app/lib/session.coffee

```diff
@@ -1,29 +1,17 @@
-storage = window.sessionStorage ? window.localStorage
+storage = window.sessionStorage
 stored = JSON.parse storage?.getItem('session_id')
 
 generateSessionID = () ->
   hash = require('hash.js')
   sha2 = hash.sha256()
   id = sha2.update("#{Math.random() * 10000 }#{Date.now()}#{Math.random() * 1000}").digest('hex')
-  ttl = fiveMinutesFromNow()
-  stored = {id, ttl}
+  stored = {id}
   try
     storage.setItem('session_id', JSON.stringify(stored))
   stored
 
 getSessionID = () ->
-  {id, ttl} = JSON.parse(storage.getItem('session_id')) ? generateSessionID()
-  if ttl < Date.now()
```

Comment:
This condition was comparing a Date() string to a Date.now() number and therefore never met. It was copied directly from PFE's codebase, so I'll be doing the same cleanup over there.